mods/ts/rules/shared-support.yaml

```diff
@@ -29,7 +29,7 @@ NAPULS:
 		Cost: 1000
 	Tooltip:
 		Name: EMP Cannon
-		Description: Disables vehicles. \nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
+		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs Tanks\n  Cannot target Aircraft
```

Comment:
In TS "tanks" is pretty iffy, because there are only like 3 "tanks" total. I'd rather go with "vehicles".